### ISSUE TITLE
Request: retain icon preferences after icon pack update

### ISSUE THREAD

**Issue by LiddersUK** [amirzaidi/Launcher3#578]
The ability to choose whether or not to use an individual icon from the in-use icon pack is cool.
But every time my preferred icon pack (lines free) is updated, I lose my personalized settings and have to set it up again.

Request that preferences are persisted after an update to the icon pack.

**Comment by LiddersUK** [comment]
Sorry, where are my manners:
Please consider this feature request.
Thanks for all your hard work Amir. This is a fantastic showcase of your abilities.

**Comment by amirzaidi** [comment]
Yes, I will absolutely fix this for the next major update, but I might wait for 9.1 before I do that. I know my icon pack implementation is messy, that's why I don't want to touch it and would rather recode it sometime soon. Unfortunately my university has started again so I don't have a lot of time, that's why I'm considering waiting until 9.1 for another restart.
